pkg/gather/clusterconfig/1-examples_test.go

```diff
@@ -0,0 +1,35 @@
+package clusterconfig
+
+import (
+	"fmt"
+)
+
+func ExampleGatherMostRecentMetrics_Test() {
```

Comment:
What is the purpose of this file ? Could it be removed ?

Reply:
Let me try to rephrase Martin here. This is used by the doc generator. In general this generator takes `GatherXY` function and then executes `ExampleGatherXY` and saves the result to the doc as sample output.